### PR TITLE
Align with nephio repo Makefile and prow config

### PR DESCRIPTION
**Pull request by johnbelamaric**

### CODE REVIEW

**Comment by johnbelamaric** [comment]
Doh

**Review by gvbalaji — APPROVED** [review]
Looks good. Approved.

File: Makefile
Comment (vjayaramrh) [inline Makefile:58]: L58 to L69 can be replaced with contents from https://github.com/nephio-project/nephio/blob/main/Makefile

**Review by vjayaramrh — CHANGES_REQUESTED** [review]
Would be great of the Makefile, .prow.yml and .golangci.yml changes from the nephio project are incorporated. THanks

**Comment by johnbelamaric** [comment]
/retitle Align with nephio repo Makefile and prow config

**Comment by johnbelamaric** [comment]
@vjayaramrh how is gosec working in the nephio repo? Make not found?

File: .prow.yaml
Comment (radoslawc) [inline .prow.yaml:17]: securego/gosec image doesn't have 'make' binary hence the error in prow job:
```
{"component":"entrypoint","error":"could not start the process: exec: \"make\": executable file not found in $PATH","file":"k8s.io/test-infra/prow/entrypoint/run.go:79","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2023-04-07T17:30:09Z"}
```

**Comment by johnbelamaric** [comment]
@radoslawc we continue to see the "lgtm restricted to collaborators". And since it's my PR, I can't lgtm it...and that means it cannot merge, since somehow I seem to be the only one that can lgtm (maybe Stephen could, he may have write access on the repo).

I CAN forcible merge it...but I won't yet...we still need to understand the gosec issue (I think it shouldn't be calling make, let me try something)

**Comment by johnbelamaric** [comment]
@radoslawc There must be something about the prow plugin config we need to change to allow any org member to LGTM

**Pull request merged**
